lnwallet/btcwallet/btcwallet.go

```diff
@@ -546,6 +546,7 @@ func minedTransactionsToDetails(
 // for an unconfirmed transaction to a transaction detail.
 func unminedTransactionsToDetail(
 	summary base.TransactionSummary,
+	chainParams *chaincfg.Params,
```

Comment:
Nit: the wrapping can be changed to the following:

```
func unminedTransactionsToDetail(summary base.TransactionSummary,
	chainParams *chaincfg.Params) (*lnwallet.TransactionDetail, error) {
```

Reply:
fixed